### PR TITLE
Mage_Directory - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
- doc blocks added/fixed
- PSR2 fixes (whitespaces, linebreaks, ...)
- fix: `Cannot use [] for read` in Mage_Directory_Model_Resource_Country_Collection`::addCountryCodeFilter() (tested)

### CODE REVIEW

File: app/code/core/Mage/Directory/Helper/Data.php [hunk 1]
```diff
@@ -107,14 +107,14 @@ public function __construct(array $args = array())
...
+     *
      * @return Mage_Directory_Model_Resource_Region_Collection
      */
-    public function getRegionCollection($countryFilter = null)
+    public function getRegionCollection()
     {
         if (!$this->_regionCollection) {
             $this->_regionCollection = Mage::getModel('directory/region')->getResourceCollection()
-                ->addCountryFilter($countryFilter)
+                ->addCountryFilter($this->getAddress()->getCountryId())
```
Comment (tmotyl) [inline app/code/core/Mage/Directory/Helper/Data.php:117]: hmm?

Comment (sreichel) [reply app/code/core/Mage/Directory/Helper/Data.php:117]: Mentioned above ....

`$countryFilter` wasn't definded. I've tested this.

**Comment by tmotyl** [comment]
please resolve the conflict

File: app/code/core/Mage/Directory/Helper/Data.php (hunk 1 above)
Comment (Flyingmana) [reply app/code/core/Mage/Directory/Helper/Data.php:117]: see https://github.com/OpenMage/magento-lts/issues/713
changed again in https://github.com/OpenMage/magento-lts/pull/1644

**Pull request merged**